ARK Server Manager/ServerSettingsControl.xaml

```diff
@@ -1152,7 +1152,12 @@
                     </Expander>
                     <Expander IsExpanded="True" Style="{DynamicResource ExpanderStyle1}">
                         <Expander.Header>
-                            <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_ChatAndNotificationsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                            <StackPanel Orientation="Horizontal">
```

Comment:
I was hoping to right align the reset button, but could not work out how to do it. But if you prefer it close to the textbox, then that it fine with me.
I am also going to disable the button if the section is collapsed.